### ISSUE TITLE
Steam input screws up gamepad support in popular titles.

### ISSUE THREAD

**Issue by AlonzoTG** [ValveSoftware/steam-for-linux#7257]
I found that for back catalog/indie games Steam basically does nothing with gamepad support so therefore it works great. 

Unfortunately if it's an AAA game or if it is popular, Steam takes it upon itself to "improve" gamepad support almost always leaving it an unusable mess or breaking gamepad support completely. If I'm lucky I am able to find a workaround and kinda get it to work. Work being the operative word there because I often have to do a lot of work on gamepad support before I can play the game at all.  In the case of Deus Ex: mankind divided on linux, I can't make it work at all these days. 

Basically, this is how gamepad support should be handled in all cases, no exceptions: 

https://www.oilystuffblog.com/single-post/2018/04/13/Oilfield-Related-Problems

**Comment by nded03** [comment]
What controller are you using?

**Comment by AlonzoTG** [comment]
Generic "Xpad". I prefer the Logitech 310 but have this "Afterglow" controller in front of me rn (the A button got squishy, left stick pulls to the left too, and unreliable so I took it off my main gaming machine, at my secondary linux machine.) They are basically the same, linux essentially ignores any differences that exist. The controler was assigened /dev/input/js0 so it should work fine... With steam input disabled, the game cannot see the gamepad, with steam input enabled, it behaves as if the gamepad is working (provides the correct gamepad button prompts) but it absolutely does not. Steaminput seems to route the secondary stick through the mouse input, (and is difficult to disable), but the gamepad input path does not work.

**Comment by etisdew** [comment]
@AlonzoTG I'm also in the same situation working on support for a wired Afterglow, did you ever find a solution?

**Comment by kisak-valve** [comment]
Hello @AlonzoTG, @etisdew, are either of you still experiencing this issue on an up to date system?

**Comment by kisak-valve** [comment]
Closing pending feedback.
